src/YarpUtilities/include/BipedalLocomotion/YarpUtilities/RosPublisher.h

```diff
@@ -0,0 +1,140 @@
+                            const BipedalLocomotion::GenericContainer::Vector<const double>::Ref jointPositions,
+                            const BipedalLocomotion::GenericContainer::Vector<const double>::Ref jointVelocities);
+    
+    /**
+     * Publish the joint states over the configured joint states topic 
+     * The joint velocities and joint efforts are set to zero.
+     * @param[in] jointList list of joints
+     * @param[in] jointPositions list of joint positions in m or rad with the same size as the joints list
+     * @param[in] jointVelocities list of joint velocities in m/s or rad/s with the same size as the joints list
+     * @param[in] jointPositions list of joint torques/forces in Nm or N with the same size as the joints list
+     */
+    bool publishJointStates(const BipedalLocomotion::GenericContainer::Vector<const std::string>::Ref jointList,
+                            const BipedalLocomotion::GenericContainer::Vector<const double>::Ref jointPositions,
+                            const BipedalLocomotion::GenericContainer::Vector<const double>::Ref jointVelocities,
+                            const BipedalLocomotion::GenericContainer::Vector<const double>::Ref jointEfforts);
+    
+    /**
+     * Publish the wrenches over the configured wrench topic associated to the frame
+     * @param[in] frameName frame at which the wrench will be expressed
+     * @param[in] wrench6d 6d wrench as force-torque in N-Nm
+     */
+    bool publishWrench(const std::string& frameName, 
+                       BipedalLocomotion::GenericContainer::Vector<const double>::Ref wrench6d);
+    
+    /**
+     * Publish transforms to the transform server
+     * @param[in] target  target frame for the transform
+     * @param[in] source source frame of the transform
+     * @param[in] transformAsVector16d 4x4 transform as a vector data
+     */
+    bool publishTransform(const std::string& target, const std::string& source, 
+                          const BipedalLocomotion::GenericContainer::Vector<const double>::Ref transformAsVector16d);
+    
+    /**
+     * configure which transform server to connect to
+     * @param[in] transformServerPort name of the transform server port
+     */
+    bool configureTransformServer(const std::string& transformServerPort);
+    
+    /**
+     * configure the topic over which joint states are published
+     * @param[in] topicName topic name      
+     */
+    bool configureJointStatePublisher(const std::string& topicName);    
+    
+    /**
+     * add or reconfigure a wrench publisher
+     * @param[in] frameName frame name
+     * @param[in] topicName topic name      
+     */
+    bool configureWrenchPublisher(const std::string& frameName, 
+                                  const std::string& topicName);
+    
+    /**
+     * remove wrench publisher
+     * @param[in] frameName frame name 
+     */
```

Comment:
It is really just my OCD speaking, but please use the first capital letter in the documentation.

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/156/commits/d57a286f4780f87c335ff0b3b91ca01443ffa069